src/System.Numerics.Tensors/pkg/System.Numerics.Tensors.pkgproj

```diff
@@ -6,10 +6,13 @@
     <MinClientVersion>2.8.6</MinClientVersion>
   </PropertyGroup>
   <ItemGroup>
-    <ProjectReference Include="..\ref\System.Numerics.Tensors.csproj">
-      <SupportedFramework>net45;wpa81;win8;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
+    <!-- We don't include the reference project for two reasons:
```

Comment:
I think that it is good to have.

Reference assemblies are smaller and so can reduce overhead during compilation (and all the other reasons why Roslyn decided to build such functionality directly into the compiler).

Reply:
No one uses it though. There are currently no plans for anyone in `corefx` to take a dependency on this library.

Reply:
I meant for inclusion in the nuget package (for external consumers), after we come up with a better fix for the docs issue (which might just involve getting it included in the intellisense package, if nothing else).

The ref project is still being validated during compilation, even if it isnt consumed from the pkgproj (it is still in the sln and still being built).

Reply:
Oh, I was assuming this was the "long term" solution (since it resolves #33692). If excluding the `ref` assembly is only temporary, then I'm OK with it.

What are we using to track the real "long term" solution?

Reply:
I'm going to log a bug indicating we should ensure S.N.T is properly documented and included in the M.Private.Intellisense package

Reply:
I think most of the info is documented here: https://github.com/dotnet/roslyn/blob/master/docs/features/refout.md

I'll see if the compiler team has any other metrics and get back to you, however.

Reply:
I see this: 

> The main purpose of the /refout option is to speed up incremental build scenarios

They went out of their way to state incremental build.  I would imagine that's for the scenario where A > B, B changes but not in a way that its reference assembly changes, so A does not need to be rebuilt.  That's less relevant for packages, since a packages are not changing (or when they do, the whole content changes, not just the impl).

Let me know if you find more data about perf as this would impact decisions that we're making elsewhere.  /cc @joperezr